packages/edit-site/src/components/global-styles/font-library-modal/context.js

```diff
@@ -346,6 +347,11 @@ function FontLibraryProvider( { children } ) {
 		getFontCollections();
 	}, [] );
 
+	const disableFontLibrary = useSelect( ( select ) => {
+		const { getEditorSettings } = select( editorStore );
+		return getEditorSettings().disableFontLibrary;
```

Comment:
I'm thinking this should be `enableFontLibrary` instead, as that seems more consistent with other editor settings (see https://github.com/WordPress/gutenberg/blob/0e48342278e3991f474759e2a537b3ad15d58b37/packages/editor/src/store/defaults.js#L27)

Reply:
Part of why naming it `disable` made sense to me was that it seems to more clearly set the expectation that the font library is inherently part of the site editor, versus explicitly having to enable it or something.

Good idea though to look at the other settings, but it doesn't look so consistent, for example [disableLayoutStyles](https://github.com/WordPress/wordpress-develop/blob/c3639812be7767e976018aece9af0a637e79d878/src/wp-includes/block-editor.php#L626).  

What do you think about naming the setting `fontLibrary` and defaulting `true`, following this convention:

https://github.com/WordPress/gutenberg/blob/0e48342278e3991f474759e2a537b3ad15d58b37/packages/block-editor/src/store/defaults.js#L142-L143

Reply:
Actually, I'm also okay with `fontLibraryEnabled` since it follows the settings that are specific to the site editor.

Reply:
Thanks for looking into it more deeply! To make it even more difficult we have both `enable[Setting]`, like

- `enableOpenverseMediaCategory`
- `enableCustomFields`

as well as `[setting]Enabled`

- `richEditingEnabled`
- `codeEditingEnabled`

I think overall I see more of the first kind (`enable[Setting]`) in editor settings, but it doesn't look like there's a standard convention to follow.

Reply:
Oh good point, I think this feels conceptually closer to the `enableOpenverseMediaCategory` and `enableCustomFields`, so let's just go with that naming pattern. Thanks!